Classes/Console/Install/Upgrade/UpgradeWizardExecutor.php

```diff
@@ -39,8 +41,13 @@ public function executeWizard(string $identifier, array $rawArguments = [], bool
 
         if ($force) {
             $closure = \Closure::bind(function () use ($upgradeWizard) {
-                /** @var DummyUpgradeWizard $upgradeWizard here to avoid annoying (and wrong) protected method inspection in PHPStorm */
-                $upgradeWizard->markWizardAsDone(0);
+                if ($upgradeWizard instanceof UpgradeWizardInterface) {
+                    // @todo you probably don't want an instance of the UpgradWizardsService here, that's why adding this here plain
+                    GeneralUtility::makeInstance(Registry::class)->set('installUpdate', $upgradeWizard->getIdentifier(), 1);
```

Comment:
that is wrong here. needs to be set to 0, as the purpose is to mark it as undone

Reply:
and the question still is, whether it needs to be the identifier or the class name, as the UpgradWizardsService is inconsistent in that regard that it is not possible to mark a wizard as undone, when using it.